src/screens/DataApi/DataCoverageTable.style.tsx

```diff
@@ -0,0 +1,77 @@
+import { TableHead, Typography } from '@material-ui/core';
+import TableCell from '@material-ui/core/TableCell';
+import TableRow from '@material-ui/core/TableRow';
+import { materialSMBreakpoint } from 'assets/theme/sizes';
+
+import { InfoIcon } from 'components/InfoTooltip/Tooltip.style';
+import styled from 'styled-components';
+
+export const TextWithTooltipContainer = styled.div`
+  ${InfoIcon} {
+    margin-left: 0.2rem;
+    vertical-align: middle;
+  }
+`;
+
+export const UnderlinedText = styled(Typography)`
+  text-decoration-line: underline;
+  text-decoration-style: dotted;
+  text-underline-position: under;
+  vertical-align: middle;
+  font-size: 14px;
+`;
+
+export const StyledTableHead = styled(TableHead)`
+  font-family: Roboto;
+  font-weight: 500;
+  font-size: 13px;
+  text-transform: uppercase;
+  background-color: white;
+`;
+
+export const MetricTableCellHeader = styled(TableCell).attrs(props => ({
+  component: 'th',
+  align: 'left',
+  scope: 'row',
+}))`
+  width: 30%;
+
+  @media (max-width: ${materialSMBreakpoint}) {
+    padding: 8px;
+    padding-left: 0px;
+    width: 20%;
+  }
+`;
```

Comment:
nit: No https://material-ui.com/api/table-head/ ? It's probably okay since you are telling it to use `th`

Reply:
So actually [it looks like](https://material-ui.com/api/table-cell/) any `TableCell`s that have a parent context of a `TableHead` (which this does) automatically turn into `th`